src/clients/ipfs.ts

```diff
@@ -40,10 +43,19 @@ export const extractHashFromURL = (urlString: string) => {
 }
 
 const init = async () => {
+
+  const ETH_INFURA_IPFS_ID = process.env.INFURA_IPFS_ID
```

Comment:
need to add to .env.example?

also, we should try make this based on a generic IPFS node rather than infura specific, so i'd use 2 env vars like:
 - IPFS_NODE_RPC_URL="https://ipfs.infura.io:5001/api/v0"
 - IPFS_NODE_AUTH_HEADER="Basic id:secret"

(with auth being optional if it's not needed)